Kluwer Arbitration.js

```diff
@@ -0,0 +1,1039 @@
+					},
+					{
+						"note": "Parties: Claimant, Société d'études et représentations navales et industrielles (Soerni) et autres<br/>Defendant, société Air Sea Broker Ltd. (ASB)"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.kluwerarbitration.com/document/kli-ka-1134507-n",
+		"items": [
+			{
+				"itemType": "statute",
+				"nameOfAct": "UNCITRAL Model Law on International Commercial Arbitration (1985, with 2006 amendments)",
+				"creators": [],
+				"dateEnacted": "2008",
+				"url": "https://www.kluwerarbitration.com/document/kli-ka-1134507-n",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [
+					{
+						"note": "Bibliographic reference: 'UNCITRAL Model Law on International Commercial Arbitration (1985, with 2006 amendments)', in Lise Bosman (ed), ICCA International Handbook on Commercial Arbitration, (© Kluwer Law International; ICCA & Kluwer Law International 2023, Supplement No. 52, June 2008), pp. 1 - 17."
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.kluwerarbitration.com/document/case/2113/35563/kli-ka-kaces-25110",
+		"items": [
+			{
+				"itemType": "case",
+				"caseName": "NJSC Naftogaz of Ukraine, PJSC State Joint Stock Company Chornomornaftogaz, PJSC Ukrgasvydobuvannya, Others. v. Russian Federation",
+				"creators": [
+					{
+						"firstName": "Anna",
+						"lastName": "Kozmenko",
+						"creatorType": "counsel"
+					},
+					{
+						"firstName": "William Ian Cornell",
+						"lastName": "Binnie",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Baiju S.",
+						"lastName": "Vasani",
+						"creatorType": "counsel"
+					},
+					{
+						"firstName": "Christopher",
+						"lastName": "Boog",
+						"creatorType": "counsel"
+					},
+					{
+						"firstName": "Elliott",
+						"lastName": "Geisinger",
+						"creatorType": "counsel"
+					},
+					{
+						"firstName": "Julie",
+						"lastName": "Raneda",
+						"creatorType": "counsel"
+					},
+					{
+						"firstName": "Charles",
+						"lastName": "Poncet",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Maja",
+						"lastName": "Stanivuković",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Erin",
+						"lastName": "Thomas",
+						"creatorType": "counsel"
+					},
+					{
+						"firstName": "Joshua B.",
+						"lastName": "Picker",
+						"creatorType": "counsel"
+					},
+					{
+						"firstName": "Marney L.",
+						"lastName": "Cheek",
+						"creatorType": "counsel"
+					}
+				],
+				"dateDecided": "12/04/2023",
```

Comment:
Dates like this should always be saved as ISO (YYYY-MM-DD) dates. Otherwise this is ambiguous, and it will be parsed as either December or April depending on the user's locale.

Reply:
Is it better to use `ZU.strToDate()` or `ZU.strToISO()`?